client/src/components/common/Card.svelte

```diff
@@ -1,14 +1,23 @@
 <script>
   import { slide } from 'svelte/transition';
-  import { deleteRecipe } from '@/utils/scripts/deleteRecipe';
+  import { deleteRecipe, togglePublish } from '@/utils/scripts/helpers';
```

Comment:
the same goes for togglePublish being in Card.svelte as it goes for deleteRecipe i commented in the previous PR